src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -260,7 +260,7 @@ ShouldUseSubqueryPushDown(Query *originalQuery, Query *rewrittenQuery)
 	 * Some unsupported join clauses in logical planner
 	 * may be supported by subquery pushdown planner.
 	 */
-	whereClauseList = WhereClauseList(rewrittenQuery->jointree);
+	whereClauseList = QualifierList(rewrittenQuery->jointree);
```

Comment:
Once we are on this, I would feel better if we also change ```WhereClauseList()``` call in ```MultiNodeTree()``` to ```QualifierList()```. 

Also consider renaming variable name to ```clauseList``` or something like that because it is not where clause list any more